content/tutorials/token-authentication.textile

```diff
@@ -244,6 +344,16 @@ blang[ruby].
 
   Try running your server again with @bundle exec ruby server.rb@ and visiting "http://localhost:4567/":http://localhost:4567/ to make sure the new static home page is being served up correctly. If you can see the static page, then it's time to set up a Realtime client that uses token authentication.
 
+blang[python].
+  Now we need to serve a static home page. Let's replace the code that responds to the @/@ route (look for @ class index: def GET(self): ... @) with the following:
+
+  ```[python]
+    html = open('index.html')
```

Comment:
To avoid ambiguity, why not include the full class def such as:
```
class index:
  def GET(self):
     html = open('index.html')
     return html.read()
```